src/services/csvUpload/load_csv_data.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
```

Comment:
Okay, so you did create this directory - nice. Just remove all the old files then